packages/teams-js/src/internal/telemetry.ts

```diff
@@ -1,4 +1,4 @@
-import { debug as registerLogger, Debugger } from 'debug';
+import { debug as registerLogger, Debugger } from 'debug/src/browser';
```

Comment:
Do we also have to care about the reference to this library from microsoft-teams-library-js\packages\teams-js\src\internal\hostToAppTelemetry.ts ?

Reply:
No, we do not as the issue only lies in the `debug` object's implementation (the named import, not the package). In `hostToAppTelemetry` we are only importing the `Debugger` interface which comes from the typings file.